packages/marshal/src/marshal.js

```diff
@@ -345,8 +345,14 @@ function assertCanBeRemotable(val) {
   assert(!Array.isArray(val), X`Arrays cannot be pass-by-remote`);
   assert(val !== null, X`null cannot be pass-by-remote`);
 
-  const keys = ownKeys(val);
+  const descs = getOwnPropertyDescriptors(val);
+  const keys = ownKeys(descs); // enumerable-and-not, string-or-Symbol
   keys.forEach(key => {
+    // @ts-ignore
+    assert(
+      !descs[key].get,
```

Comment:
```suggestion
      !('get' in descs[/** type{string}*/ (key)]),
```
Somewhere recently I just came across this cast in some of our code. I did the @ts-ignore trick myself, only just recently but before I saw this. If this cast is the best way to do it, I should track down my `@ts-ignore`s for this and fix them,

It makes NO SENSE for typescript to prohibit indexing by symbols. Using symbols as property names is the use case for which symbols were introduced! And you can only use them as property names by indexing syntax. Attn @michaelfig is there something about this prohibition that I'm not getting?

Reply:
I tried that inline `type` and it didn't seem to work. I broke down and moved the `// @ts-ignore` into the middle of the multi-line `assert()` clause and that did the trick.

Reply:
> It makes NO SENSE for typescript to prohibit indexing by symbols. Using symbols as property names is the use case for which symbols were introduced! And you can only use them as property names by indexing syntax. Attn @michaelfig is there something about this prohibition that I'm not getting?


TL;DR: This is a bug in Typescript that was noticed in 2015 but has never been corrected.  It has a "help wanted" tag on it.

You can have some fun reading at https://github.com/microsoft/TypeScript/issues/1863 (100+ comments).